api/zulip/__init__.py

```diff
@@ -188,6 +194,21 @@ def __init__(self, email=None, api_key=None, config_file=None,
 
         if config_file is None:
             config_file = get_default_config_filename()
+
+        # Fill values from Environment Variables if not available in Constructor
+        if api_key is None:
+            api_key = get_env_variable("ZULIP_API_KEY")
+        if email is None:
+            email = get_env_variable("ZULIP_EMAIL")
+        if site is None:
+            site = get_env_variable("ZULIP_SITE")
+        if client_cert is None:
+            client_cert = get_env_variable("ZULIP_CERT")
+        if client_cert_key is None:
+            client_cert_key = get_env_variable("ZULIP_KEY")
+        if cert_bundle is None:
+            cert_bundle = get_env_variable("ZULIP_BUNDLE")
```

Comment:
Cool, I think I agree this is the right precedence -- command line over environment over config file.